pandas/_libs/tslibs/c_timestamp.pyx

```diff
@@ -320,7 +320,7 @@ cdef class _Timestamp(datetime):
         cdef:
             int64_t val
             dict kwds
-            int8_t out[1]
```

Comment:
what does it complain about here?  we put some effort into getting `ndarray` out of this file

Reply:
Warning was about uninitialized value. But this change failed the tests on type. It's now a memoryview of a boolean. I think the bad bit is the explicit len.

Reply:
... which passes locally on py37 and py35, but the CI consistently fails. So skipping this now.